### PR TITLE
docs(filemanager): Add dummy file to OnDownload example

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: components/filemanager/events.md [hunk 1]
```diff
@@ -413,14 +413,14 @@ The `ViewChanged` event fires when the user toggles between the [two FileManager
 
         if (parentDirectory != null)
         {
-            // simulate add in file system
+            // Simulate add to the file system
```
Comment (ikoevska) [inline components/filemanager/events.md:416]: I recommend using ending punctuation consistently. Either all code comments should end with a period (or the semi-colon in some of the instances), or none.

File: components/filemanager/events.md [hunk 2]
```diff
@@ -512,7 +518,7 @@ The `ViewChanged` event fires when the user toggles between the [two FileManager
 
     private void OnSelect(IEnumerable<FlatFileEntry> selectedFiles)
     {
-        //update the view-model
+        // Update the view-model
```
Comment (ikoevska) [inline components/filemanager/events.md:521]:
```suggestion
        // Update the view model.
```

**Review by ikoevska — APPROVED** [review]

File: components/filemanager/events.md [hunk 3]
```diff
@@ -413,14 +413,14 @@ The `ViewChanged` event fires when the user toggles between the [two FileManager
...
-            // simulate add in file system
+            // Simulate add to the file system
             newFolder.ParentId = parentDirectory.Id;
             Files.Add(newFolder);
             parentDirectory.HasDirectories = Files.Count(x => x.ParentId == parentDirectory.Id) > 0;
         }
         else
         {
-            // create a folder in the root dir
+            // Create a folder in the root directory
```
Comment (dimodi) [inline components/filemanager/events.md:423]:
```suggestion
            // Create a folder in the root directory.
```

File: components/filemanager/events.md (hunk 1 above)
Comment (dimodi) [inline components/filemanager/events.md:416]:
```suggestion
            // Simulate add to the file system.
```

**Pull request merged**